src/firefly/js/util/VOAnalyzer.js

```diff
@@ -0,0 +1,337 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import {get, has, isEmpty} from 'lodash';
+import Enum from 'enum';
+import {getColumn, getColumnIdx} from './../tables/TableUtil.js';
+import {MetaConst} from '../data/MetaConst.js';
+import {CoordinateSys} from '../visualize/CoordSys.js';
+
+const UCDCoord = new Enum(['eq', 'ecliptic', 'galactic']);
+
+//const pObs = 'obscore:';
+const ObsTapColName = 0;
```

Comment:
Minor: it took me some time to trace the meaning of this field. It might be easier, if it's closer to ObsTapMandColumns definition or has an explanation comment.